<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoveCameraDialog</class>
 <widget class="QDialog" name="MoveCameraDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>248</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kameraposition setzten</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Look-At</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QLineEdit" name="lookAtZ"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <family>Sans Serif</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>z</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lookAtX"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <family>Sans Serif</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>y (Höhe)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lookAtY"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <family>Sans Serif</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>x</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Kameraposition</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <family>Sans Serif</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>y (Höhe)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="camX"/>
     </item>
     <item row="1" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_12">
       <property name="font">
        <font>
         <family>Sans Serif</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>x</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="camY"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <family>Sans Serif</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>z</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="camZ"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="doButton">
     <property name="text">
      <string>Durchführen</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
